### ISSUE TITLE
`--cors-allow-origins` flag has no effect

### ISSUE THREAD

**Issue by mr-moon** [joseluisq/static-web-server#86]
**Describe the bug**
Starting a server with `--cors-allow-origins "*"` does not send any CORS headers.

**To Reproduce**
``docker run --rm -it -p '8888:8888' -v `pwd`:/web joseluisq/static-web-server:2.6 --cors-allow-origins "*" -p 8888 -d /web``
serve any `json`, or `log` text file.

**Expected behavior**
CORS headers are present.

**Logs/Screenshots**
```
$ curl -I http://localhost:8888/73f944bde7bb0b9093e5f1fb9d01ad1f.json
HTTP/1.1 200 OK
content-length: 548
content-type: application/json
accept-ranges: bytes
last-modified: Mon, 28 Feb 2022 14:19:29 GMT
cache-control: public, max-age=3600
date: Mon, 28 Feb 2022 14:20:45 GMT
```

**Environment and Specs**
 - static-web-server: 2.5, 2.6
 - OS: docker image 
 - Docker: 20.10.12
 - Client: curl, but doesn't matter

**Comment by joseluisq** [comment]
Yeah, headers are not properly assigned. So a fix will come soon.

**Comment by joseluisq** [comment]
Please have a look at https://github.com/joseluisq/static-web-server/pull/87#issuecomment-1057467997 for more details.

**Comment by joseluisq** [comment]
> ```
> $ curl -I http://localhost:8888/73f944bde7bb0b9093e5f1fb9d01ad1f.json
> HTTP/1.1 200 OK
> content-length: 548
> content-type: application/json
> accept-ranges: bytes
> last-modified: Mon, 28 Feb 2022 14:19:29 GMT
> cache-control: public, max-age=3600
> date: Mon, 28 Feb 2022 14:20:45 GMT
> ```

BTW in your example you have to provide the `Origin` header in order to try to get access the resource in a cross-origin manner.
Take also a look at the PR #87 description.

**Comment by joseluisq** [comment]
For testing please use the latest development Docker images either `joseluisq/static-web-server:devel-alpine` or `joseluisq/static-web-server:devel`

**Comment by joseluisq** [comment]
Fixed on [v2.7.0](https://github.com/joseluisq/static-web-server/releases/tag/v2.7.0)
